types/result.go

```diff
@@ -39,19 +39,42 @@ func (res Result) IsOK() bool {
 	return res.Code.IsOK()
 }
 
-// Is a version of TxResponse where the tags are StringTags rather than []byte tags
+// IndexedABCILogs represents a slice of IndexedABCILog.
+type IndexedABCILogs []IndexedABCILog
+
+// IndexedABCILog defines a structure containing an indexed tx ABCI message log.
+type IndexedABCILog struct {
```

Comment:
Considering that:
`IndexedABCILog defines a structure containing an indexed tx ABCI message log.`

wouldn't it be better to call this type `ABCIMessageLog` or `ABCIMessageLogEntry`?

Reply:
I like `ABCIMessageLog`. Will update.